### PR TITLE
Adding links for donations

### PR DESCRIPTION
**Pull request by mrshirts**
Please check over the text to see if it's phrased.  My local jekyll environment is broken, so I need to fix that and test how it looks, but please comment on the content, organization and placement of the material within the overall website.  The link from the main page is described here: https://github.com/livecomsjournal/journal_information/pull/79

I figure it is important to make the financing clear, so included that section.

### CODE REVIEW

File: _about_livecoms/04_donations.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  We very much appreciate financial donations to LiveCoMS! Donations
+  are made to the CU Foundation of University of Colorado Boulder, the
+  official publisher of the journal, to a fund exclusively for use of
+  LiveCoMS and controlled by the managing editors. The CU Foundation
+  is a 501(c)(3) charitable organization, and donations are thus
+  usually tax-deductible.
+ 
+Follow the link felow to donate:
+
+  <p><a href="https://giving.cu.edu/fund/living-journal-computational-molecular-science-support-fund" class="btn ">Donate Now to LiveCoMS!</a></p>
```
Comment (davidlmobley) [inline _about_livecoms/04_donations.md:21]: Is the `class="btn "` tag functional?

File: _about_livecoms/04_donations.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+You can also support LiveCoMS by:
+  * Contributing to LiveCoMS articles.
+  * Expressing your interest in becoming a reviewer ([early career](/about/earlycareer) or otherwise).
```
Comment (davidlmobley) [inline _about_livecoms/04_donations.md:25]: Maybe note that currently we have a large backlog of such reviewers? I don't think we want to encourage a deluge of them at present, becuase someone has to track and already we have a lot more than we're using.

File: _about_livecoms/04_donations.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+are:
+
+* $1200/year for journal hosting by [Scholastica](http://www.scholasticahq.com).
+* $250/year for reviewer services by [Scholastica](http://www.scholasticahq.com).
+* $504/year for Google Suites hosting of email and collaborative office software
+
+The main sources of income for LiveCoMS are donations and article
+charges. We in particular appreciate seed funding provided by [CU
+Boulder Libraries](https://www.colorado.edu/libraries/), as well as continuing in-kind contributions by
+managing DOI submissions and journal metadata. Article charges are
```
Comment (davidlmobley) [inline _about_livecoms/04_donations.md:45]: Maybe "continuing support in terms of management of DOIs and journal metadata" would be clearer; "in-kind contributions ... by..." sounds awkward.

File: _about_livecoms/04_donations.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+$100 for each article upon submission, from which we receive about
+$80 after charges from Scholastica and processing fees.
+
+If we raise more money through donations than cover our costs, we will work to:
+  * Hire web developers and designers to support better .tex templates and improvements to the website
+  * Investigate advertising options to better raise awareness of the journal and its purposes. 
+  * Hire copy editors to assist in final journal submissions
+  * Waiving submission fees in part (based on need) or in whole.
```
Comment (davidlmobley) [inline _about_livecoms/04_donations.md:53]: "Waiving" -> "Waive"

**Review by davidlmobley — APPROVED** [review]
Basically looks good, just a few minor improvements suggested.

File: _about_livecoms/04_donations.md (hunk 2 above)
Comment (dmzuckerman) [reply _about_livecoms/04_donations.md:25]: I would hesitate to discourage potential reviwers.  There's not much harm in having a bigger list - to us or the reviewers ... we don't promise anything.

Comment (mrshirts) [reply _about_livecoms/04_donations.md:25]: agreed with dan.

File: _about_livecoms/04_donations.md (hunk 1 above)
Comment (mrshirts) [reply _about_livecoms/04_donations.md:21]: supposed to be may need to test in live version.

File: _about_livecoms/04_donations.md (hunk 3 above)
Comment (mrshirts) [reply _about_livecoms/04_donations.md]: Done.

File: _about_livecoms/04_donations.md (hunk 4 above)
Comment (mrshirts) [reply _about_livecoms/04_donations.md]: Done.

**Pull request merged**